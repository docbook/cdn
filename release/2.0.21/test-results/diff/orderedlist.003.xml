<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <p deltaxml:deltaV2="A">Test list continuation. And cross-references;
item “2”: <a href="#i2">2</a> and item “4”: <a href="#i4">D</a>.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A!=B"><deltaxml:attributes deltaxml:deltaV2="A!=B"><dxa:class deltaxml:deltaV2="A!=B"><deltaxml:attributeValue deltaxml:deltaV2="A">orderedlist</deltaxml:attributeValue><deltaxml:attributeValue deltaxml:deltaV2="B">content</deltaxml:attributeValue></dxa:class></deltaxml:attributes>
            <ol deltaxml:deltaV2="A" style="list-style: decimal;">
               <li>First item</li>
               <li id="i2">Second item</li>
               <li>Third item</li>
            </ol><p deltaxml:deltaV2="B">Test list continuation. And cross-references;
item “2”: <a href="#i2">2</a> and item “4”: <a href="#i4">D</a>.</p><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" class="orderedlist">
               <ol style="list-style: decimal;">
                  <li>First item</li>
                  <li id="i2">Second item</li>
                  <li>Third item</li>
               </ol>
            </div><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="B">And then a break occurs...a break in my sanity, apparently,
because changing the numeration style across a “continues” break seems
really stupid. But this a unit test, after all. Stupid is
allowed.</p><deltaxml:textGroup deltaxml:deltaV2="B"><deltaxml:text deltaxml:deltaV2="B">
            </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="B" class="orderedlist">
               <ol start="4" style="list-style: upper-alpha;">
                  <li id="i4">Fourth item</li>
                  <li>Fifth item</li>
                  <li>Sixth item</li>
               </ol>
            </div>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><p deltaxml:deltaV2="A">And then a break occurs...a break in my sanity, apparently,
because changing the numeration style across a “continues” break seems
really stupid. But this a unit test, after all. Stupid is
allowed.</p><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A" class="orderedlist">
            <ol start="4" style="list-style: upper-alpha;">
               <li id="i4">Fourth item</li>
               <li>Fifth item</li>
               <li>Sixth item</li>
            </ol>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>