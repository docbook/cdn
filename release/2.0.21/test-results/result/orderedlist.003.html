<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for orderedlist.003.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: orderedlist.003.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">8</span> differences)</th><th>Actual (<span class="adiff">8</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;p&gt;Test list continuation. And cross-references;
item “2”: <span class="tag">&lt;a <span class="att">href="#i2"</span>&gt;2&lt;/a&gt;</span> and item “4”: <span class="tag">&lt;a <span class="att">href="#i4"</span>&gt;D&lt;/a&gt;</span>.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">orderedlist</span>"</span>&gt;
            <span class="tag diff del">&lt;ol <span class="att">style="list-style: decimal;"</span>&gt;
               <span class="tag">&lt;li&gt;First item&lt;/li&gt;</span>
               <span class="tag">&lt;li <span class="att">id="i2"</span>&gt;Second item&lt;/li&gt;</span>
               <span class="tag">&lt;li&gt;Third item&lt;/li&gt;</span>
            &lt;/ol&gt;</span>
         &lt;/div&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;p&gt;And then a break occurs...a break in my sanity, apparently,
because changing the numeration style across a “continues” break seems
really stupid. But this a unit test, after all. Stupid is
allowed.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;div <span class="att">class="orderedlist"</span>&gt;
            <span class="tag">&lt;ol <span class="att">start="4"</span> <span class="att">style="list-style: upper-alpha;"</span>&gt;
               <span class="tag">&lt;li <span class="att">id="i4"</span>&gt;Fourth item&lt;/li&gt;</span>
               <span class="tag">&lt;li&gt;Fifth item&lt;/li&gt;</span>
               <span class="tag">&lt;li&gt;Sixth item&lt;/li&gt;</span>
            &lt;/ol&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff">&lt;div <span class="att">class="<span class="att attchg">content</span>"</span>&gt;
            <span class="tag diff add">&lt;p&gt;Test list continuation. And cross-references;
item “2”: <span class="tag">&lt;a <span class="att">href="#i2"</span>&gt;2&lt;/a&gt;</span> and item “4”: <span class="tag">&lt;a <span class="att">href="#i4"</span>&gt;D&lt;/a&gt;</span>.&lt;/p&gt;</span><span class="diff txtchg">
            </span><span class="tag diff add">&lt;div <span class="att">class="orderedlist"</span>&gt;
               <span class="tag">&lt;ol <span class="att">style="list-style: decimal;"</span>&gt;
                  <span class="tag">&lt;li&gt;First item&lt;/li&gt;</span>
                  <span class="tag">&lt;li <span class="att">id="i2"</span>&gt;Second item&lt;/li&gt;</span>
                  <span class="tag">&lt;li&gt;Third item&lt;/li&gt;</span>
               &lt;/ol&gt;</span>
            &lt;/div&gt;</span><span class="diff txtchg">
            </span><span class="tag diff add">&lt;p&gt;And then a break occurs...a break in my sanity, apparently,
because changing the numeration style across a “continues” break seems
really stupid. But this a unit test, after all. Stupid is
allowed.&lt;/p&gt;</span><span class="diff txtchg">
            </span><span class="tag diff add">&lt;div <span class="att">class="orderedlist"</span>&gt;
               <span class="tag">&lt;ol <span class="att">start="4"</span> <span class="att">style="list-style: upper-alpha;"</span>&gt;
                  <span class="tag">&lt;li <span class="att">id="i4"</span>&gt;Fourth item&lt;/li&gt;</span>
                  <span class="tag">&lt;li&gt;Fifth item&lt;/li&gt;</span>
                  <span class="tag">&lt;li&gt;Sixth item&lt;/li&gt;</span>
               &lt;/ol&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>Test list continuation. And cross-references;
item “2”: <a href="#i2">2</a> and item “4”: <a href="#i4">D</a>.</p><div class="orderedlist"><ol style="list-style: decimal;"><li>First item</li><li id="i2">Second item</li><li>Third item</li></ol></div><p>And then a break occurs...a break in my sanity, apparently,
because changing the numeration style across a “continues” break seems
really stupid. But this a unit test, after all. Stupid is
allowed.</p><div class="orderedlist"><ol start="4" style="list-style: upper-alpha;"><li id="i4">Fourth item</li><li>Fifth item</li><li>Sixth item</li></ol></div></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="content"><p>Test list continuation. And cross-references;
item “2”: <a href="#i2">2</a> and item “4”: <a href="#i4">D</a>.</p><div class="orderedlist"><ol style="list-style: decimal;"><li>First item</li><li id="i2">Second item</li><li>Third item</li></ol></div><p>And then a break occurs...a break in my sanity, apparently,
because changing the numeration style across a “continues” break seems
really stupid. But this a unit test, after all. Stupid is
allowed.</p><div class="orderedlist"><ol start="4" style="list-style: upper-alpha;"><li id="i4">Fourth item</li><li>Fifth item</li><li>Sixth item</li></ol></div></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/orderedlist.003.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>