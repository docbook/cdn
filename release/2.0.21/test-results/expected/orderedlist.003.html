<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>Test list continuation. And cross-references;
item “2”: <a href="#i2">2</a> and item “4”: <a href="#i4">D</a>.</p><div class="orderedlist"><ol style="list-style: decimal;"><li>First item</li><li id="i2">Second item</li><li>Third item</li></ol></div><p>And then a break occurs...a break in my sanity, apparently,
because changing the numeration style across a “continues” break seems
really stupid. But this a unit test, after all. Stupid is
allowed.</p><div class="orderedlist"><ol start="4" style="list-style: upper-alpha;"><li id="i4">Fourth item</li><li>Fifth item</li><li>Sixth item</li></ol></div></article><script src="../../resources/js/prism.js"></script></body></html>